src/main/java/com/epam/controller/UserRestController.java

```diff
@@ -8,6 +8,7 @@
 
 import java.util.List;
 
+@CrossOrigin
```

Comment:
Please, dont mix front & dev changes in one pr.

Reply:
And please, remember main rule. If u copypaste something, and then find a better solution, please, check, that u deleted all useless files (which were copypasted).

Reply:
And, of course, try to write more code by urself.
Fix "no new line at end of file error"